lib/api/webSockets.js

```diff
@@ -54,6 +57,15 @@ function connect(connectionOps = {}) {
 			);
 		}
 
+		if (content.type === 'console' && content.data) {
+			const method = {
+				info: 'info',
+				log: 'log',
+			}[content.subtype] || 'debug';
+
+			return logger[method](content.data.join('\n'));
```

Comment:
We need to mark the app logs somehow so that they are easily distinguishable from the API logs. 
We now use |A| for marking assertions and |E| for evaluations, Perhaps we should use |L| for for app logs (maybe also with a different color)